.github/workflows/firestore.yml

```diff
@@ -150,3 +150,24 @@ jobs:
         scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseFirestore.podspec \
             --platforms=ios \
             --allow-warnings
+
+  quickstart:
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Bundler
```

Comment:
Just a thought about keeping our scripts CI agnostic. I wonder will it be easier to port to another CI if we create a single script to run all steps so we less rely on a CI specific config?

Reply:
The secrets management is very CI specific, so I wasn't able to come up with a scripting solution that wouldn't add complexity.

Reply:
While I'm generally supportive of keeping these scripts CI agnostic, secret retrieval is inherently CI specific.

I see this as:
  * Setup (bundler then quickstart)
  * Install secrets  <- CI specific
  * Run tests

We could probably stand to just call `setup_bundler.sh` from `setup_quickstart.sh` so that's just one step but otherwise I think this is factored pretty well.